tests/mutable_deck.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import print_function
+import unittest
+from sunbeam import parse_deck, Deck, repeat, default
+from textwrap import dedent
+
+class TestParse(unittest.TestCase):
+
+    def test_deck(self):
+        opm_deck = parse_deck(dedent("""\
+        START             -- 0
+          10 MAI 2007 /
+        RUNSPEC
+
+        DIMENS
+          2 2 1 /
+        GRID
+        DX
+          4*0.25 /
+        DY
+          4*0.25 /
+        DZ
+          4*0.25 /
+        TOPS
+          4*0.25 /
+        REGIONS
+        OPERNUM
+          3 3 1 2 /
+        FIPNUM
+          1 1 2 3 /
+        PARALLEL
+          1 "DISTRIBUTED" /
+        GRID
+        GRID
+        GRID
+        """))
+        deck = Deck(opm_deck)
+        print(deck['RUNSPEC'][0][0:1])
+        deck['RUNSPEC'][0][1:1] = [
+            ('NOSIM',),
+            ('OPTIONS', [85 * default, 1]),
+            ('OPTIONS', [231 * default, default, 1]),
+            ('OPTIONS', [default]*100 + [ i for i in range(1,10) ] + ['CHAR_DATA'])
+        ]
+        deck['GRID'][0][1:1] = ('JENS', [20 * repeat(1), 2], ['JENS', 'KRISTIAN'], [])
+        deck['PARALLEL'] = None
+        deck['GRID'][1:][1:1] = ('INSERTED',)
+        deck['GRID'][2:] = ('SWAPPED',)
```

Comment:
Sorry, but I don't see this as a viable API.  Do you to go to our users and tell them that _"the way to use this is simply to write `deck['GRID'][0][1:1] = ('JENS', [20 * repeat(1), 2], ['JENS', 'KRISTIAN'], [])`"_ ?

Design the API before you write any code.  You can write the API specifications in the PR body.